client/my-sites/stats/stats-site-overview/index.jsx

```diff
@@ -0,0 +1,107 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import SiteIcon from 'components/site-icon';
+import observe from 'lib/mixins/data-observe';
+import route from 'lib/route';
+import Card from 'components/card';
+import Gridicon from 'components/gridicon';
+
+export default React.createClass( {
+	displayName: 'StatsSiteOverview',
+
+	proptypes: {
+		site: PropTypes.object.isRequired,
+		summaryData: PropTypes.object.isRequired,
+		path: PropTypes.string.isRequired
+	},
+
+	mixins: [ observe( 'summaryData' ) ],
+
+	ensureValue( value ) {
+		if ( value || value === 0 ) {
+			return this.numberFormat( value );
+		}
+
+		// If no value present, return en-dash
+		return String.fromCharCode( 8211 );
+	},
+
+	isValueLow( value ) {
+		return ! value || 0 === value;
+	},
+
+	render() {
+		const { site, path, summaryData, insights } = this.props;
+		const { views, visitors, likes, comments } = summaryData.data;
+		const siteStatsPath = [ path, site.slug ].join( '/' );
+		let headerPath = siteStatsPath;
+		let title;
+		let icon;
+
+		if ( insights ) {
+			title = this.translate( 'Today\'s Stats' );
+		} else {
+			title = site.title;
+			icon = (
+				<div className="module-header__site-icon">
+					<SiteIcon site={ site } size={ 24 } />
+				</div>
+			);
+
+			headerPath = route.getStatsDefaultSitePage( site.slug );
+		}
+
+		return (
+			<Card key={ site.ID } className="stats__overview stats-module is-site-overview">
+				<div className="module-header">
```

Comment:
Meaning the Card should have `.stats-site-overview` and the modules should be `.stats-site-overview__module-header` or create a `stats-module` component.